cypress/e2e/example.cy.js

```diff
@@ -2,50 +2,70 @@ import { faker } from '@faker-js/faker';
 /// <reference types='cypress' />
 
 describe('Bank app', () => {
-  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
-  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
-  const balance = depositAmount - withdrawAmount;
-  const user = 'Harry Potter';
-  const accountNumber = '1004';
+  const username = 'Hermoine Granger';
+  const account = '1001';
+  const accountBalance = 5096;
+  const depositAmount = faker.number.int(5000);
```

Comment:
The `faker.number.int` method is used incorrectly. It should be `faker.number.int({ max: 5000 })` to generate a random integer up to 5000.